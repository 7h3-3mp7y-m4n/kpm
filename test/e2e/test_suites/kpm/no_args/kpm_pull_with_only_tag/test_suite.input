kpm pull --tag v0.0.1