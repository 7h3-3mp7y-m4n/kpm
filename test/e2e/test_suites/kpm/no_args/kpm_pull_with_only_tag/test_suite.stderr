kpm: oci url or package name must be specified.
kpm: failed to pull kcl package.
